src/pages/edit_resource/KalturaRegistration.tsx

```diff
@@ -119,38 +186,74 @@ const KalturaRegistration: FC<KalturaRegistrationProps> = ({ expanded, onChange,
         aria-labelledby={FormDialogTitleId}>
         <DialogTitle id={FormDialogTitleId}>{t('kaltura.my_resources')}</DialogTitle>
         <StyledDialogContent>
-          {kalturaResources && !busyGettingKalturaResources && (
-            <div ref={startOfList}>
-              <Typography variant="h3">{t('common.result')}</Typography>
-              <Typography>
-                {kalturaResources?.length > 0 ? (
-                  <>
-                    {`${t('common.showing')} ${firstItemOnPage + 1}-${lastItemOnPage} ${t('common.of').toLowerCase()} ${
-                      kalturaResources?.length
-                    }.`}
-                  </>
-                ) : (
-                  <>{t('dashboard.search_result_no_hits')}</>
-                )}
-              </Typography>
-              <Typography>{t('kaltura.choose_a_resource')}.</Typography>
-            </div>
-          )}
-          <StyledList>
+          <StyledResultList ref={startOfList}>
+            {filteredKalturaResources && kalturaResources && !busyGettingKalturaResources && (
+              <StyledFullWidth>
+                <StyledGridForFilters container justifyContent="space-between">
+                  <Grid item md={7} xs={12}>
+                    <StyledFilterBoxWrapper>
+                      <Typography display="inline" variant="body1">
+                        {t('kaltura.fill_filter_box')}:
+                      </Typography>
```

Comment:
Her mangler det noen UU ting for å få denne labelen innafor.

Reply:
håper løsningen med label holder

Reply:
StyledTextFieldWithMargin mangler id="filter-text-box", ellers ser det greit ut